saleor/graphql/webhook/mutations.py

```diff
@@ -0,0 +1,125 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ...webhook import models
+from ..core.mutations import ModelDeleteMutation, ModelMutation
+from .types import Webhook
+
+
+class WebhookSubscribeInput(graphene.InputObjectType):
+    target_url = graphene.String(description="The url to receive the payload")
```

Comment:
One would want to `secretKey` field (255 chars), blank by default to allow Saleor to send a HMAC signature header of the payload when sending the webhook event, c.f. the related issue + GH repositories are good examples.

This allows to verify the payload is actually coming from Saleor and not from somebody else by calculating the HMAC of the payload against the shared secret key.

Good reference https://developer.github.com/webhooks/securing/
Example by me https://github.com/NyanKiyoshi/pytest-django-queries-bot/blob/master/github/security/github.go

Reply:
@NyanKiyoshi I know about this and about multiple events. And this is a reason why this PR is in the draft :) I just wanted to have a draft of API structure.